README.md

```diff
@@ -1,52 +1,64 @@
-# Docker-OSX
-## [Follow @sickcodes on Twitter](https://twitter.com/sickcodes)
+# Docker-OSX 
 
-![Running mac osx in a docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
+![Running Mac OS X in a Docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
 
-Run Mac in a Docker container! Run near native OSX-KVM in Docker! X11 Forwarding! iMessage security research!
+Run OSX-KVM in Docker with near-native performance! X11 Forwarding! iMessage security research!
 
-Author: Sick.Codes https://sick.codes/ & https://twitter.com/sickcodes
+## Author
 
-Documentation: everything is on this page!
+This project is maintained by [Sick.Codes](https://sick.codes/) [(Twitter)](https://twitter.com/sickcodes)
```

Comment:
## [Follow @sickcodes on Twitter](https://twitter.com/sickcodes)

Reply:
It takes up a lot of screen space.

How about:

# Docker-OSX · [Follow @sickcodes on Twitter]